src/components/async-download-pdf/index.js

```diff
@@ -2,16 +2,17 @@
 import * as React from 'react'
 import {asyncComponent} from 'react-async-component'
 import DownloadPDF from './DownloadPDF'
-import {Spinning, Toast} from 'grommet'
-import {translate} from 'react-i18next'
+import {withTranslation} from 'react-i18next'
 
+// TODO Toast for error
 const ErrorNotification = ({t}) => (
-  <Toast status="critical">{t('unknownError')}</Toast>
+  <alert>{t('unknownError')}</alert>
 )
 
+// TODO Add spinner back for loading component
 const asyncDownload: typeof DownloadPDF = asyncComponent({
-  ErrorComponent: translate()(ErrorNotification),
-  LoadingComponent: Spinning,
+  ErrorComponent: withTranslation()(ErrorNotification),
+  LoadingComponent: () => (<div>Please wait...</div>),
```

Comment:
Looks like it also doesn't have a spinner

Reply:
Yeah, Grommet v2 doesn't have one. We could add a lightweight dependency on another package to add one, perhaps this? https://github.com/mhnpd/react-loader-spinner

Reply:
Looks like it will work!